taf/docker-compose-taf-no-secty.yml

```diff
@@ -403,7 +403,7 @@ services:
     - no-new-privileges:true
     user: 2002:2001
   device-virtual:
-    command: -cp=consul.http://edgex-core-consul:8500 --registry --confdir=CONF_DIR_PLACE_HOLDER
+    command: -cp=consul.http://edgex-core-consul:8500 --registry --confdir=CONFIG_DIR_PLACE_HOLDER
```

Comment:
Should be updated to configdir for device-services?

Reply:
Yes